layouts/shortcodes/discordInvite.html

```diff
@@ -0,0 +1,25 @@
+<!--
```

Comment:
I think we should replace this with a hugo comment:

```html
{{/* long
multiline comment here */}}
```

Reply:
Updated in latest commit. Still learning hugo syntax so thanks for the recommendation.